### DISCUSSION TITLE
Hardcoded ff7_en.exe check

### DISCUSSION THREAD

**Discussion by Bl4ckh34d** [tsunamods-codes/7th-Heaven#239] [Q&A]
ff7_en.exe check is hardcoded to the English version.
The entire program doesn't work with ff7_de.exe or any other language besides English.

Please let me know if you need any additional information.
I tried running 7th Heaven with the German version of the game and the spanish version of the game but both failed when checking the game executable. It appears that ff7_en.exe is hardcoded

lol: https://www.reddit.com/r/USdefaultism/

**Comment by julianxhokaxhiu** [comment]
7th Heaven is meant to work only on the English edition of the game and that is purpose to simplify the complexity behind mod development. This is on purpose and is not a bug. Hope this helps.
